libcore/Enums.vala

```diff
@@ -253,4 +253,41 @@ namespace Files {
             }
         }
     }
+
+    // Matches settings enum for format of datetimes in view
+    public enum DateFormatMode {
+        ISO,
+        LOCALE,
+        INFORMAL;
+
+        public string to_string () {
+            switch (this) {
+                case ISO:
+                    ///TRANSLATORS "ISO" is the acronym for "International Standards Organisation"
+                    return _("ISO");
+                case LOCALE:
+                    return _("Locale");
+                case INFORMAL:
+                    return _("Informal");
+                default:
+                    assert_not_reached ();
+            }
+        }
+
+        public static DateFormatMode from_string (string format) {
```

Comment:
It looks like this method is not called anywhere, is this expected?

Reply:
Ah, this is because the setting was originally a string but its now an enum so this is not needed any longer. I'll remove.